src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,7 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
```

Comment:
Ensure that the `user` prop is passed correctly to the `UserInfo` component. It should be an object containing `name` and `email` properties.